cranelift/filetests/filetests/runtests/fence.clif

```diff
@@ -0,0 +1,17 @@
+test interpret
+test run
+target aarch64
+target s390x
+target x86_64
+target riscv64
+
+; There isn't a whole lot we can check here
+
+function %fence() -> i8 {
+block0:
+    fence
+
+    v0 = iconst.i8 0
+    return v0
+}
+; run: %fence() == 0
```

Comment:
I guess the only thing we're checking is that compiling and executing a `fence` doesn't crash, which is good to check even if it isn't very exciting. Given that, do we gain much by returning a value?

```suggestion
; Check that the fence instruction doesn't crash. Testing anything else would
; require multiple threads, which requires a runtime like Wasmtime.

function %fence() {
block0:
    fence
    return
}
; run: %fence()
```

Reply:
Unfortunately we don't yet support having a run test that just *runs* we always need a comparison operator.

Reply:
Ah, I see. Okay, I suggest taking my version of the comment, but the test itself is fine then.